### PR TITLE
Nabil

### PR DESCRIPTION
**Pull request by hyperov**
Please review this, @casey-collab

### CODE REVIEW

File: arabic/salamoAlikom.md [hunk 1]
```diff
@@ -15,3 +15,7 @@
 [وظيفة](Job/Job.md)
 
 [أريد أن أحصل على Nanodegree](https://www.udacity.com/nanodegree)
+
+ [online](earn/learn.md) عايز اعرف اسماء مواقع  كورسات
```
Comment (casey-collab) [inline arabic/salamoAlikom.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: arabic/salamoAlikom.md [hunk 2]
```diff
@@ -15,3 +15,7 @@
 [وظيفة](Job/Job.md)
 
 [أريد أن أحصل على Nanodegree](https://www.udacity.com/nanodegree)
+
+ [online](earn/learn.md) عايز اعرف اسماء مواقع  كورسات  
+
+ [freelancer](earn/learn.md) عايز اعرف اسماء مواقع  كورسات فرى لانس
```
Comment (casey-collab) [inline arabic/salamoAlikom.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by hyperov** [comment]
i made changes ,@casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**